webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -224,7 +253,9 @@ class LicenseeList extends Vue {
     }
 
     // Match pageChange() @Prop signature from /components/Lists/Pagination/Pagination.ts
-    async paginationChange({ prevNext }: PageChangeConfig) {
+    async paginationChange({ firstIndex, prevNext }: PageChangeConfig) {
+        const isInitialInProgress = Boolean(firstIndex === 0 && prevNext === 0);
```

Comment:
If not intentional, can we change to-

```suggestion
        const isInitialInProgress = firstIndex === 0 && prevNext === 0;
```

Reply:
Updated